working-with-openai/open-ai-api-models.md

```diff
@@ -0,0 +1,79 @@
+# Introduction to OpenAI API Models
```

Comment:
To help differentiate lesson titles, even though we aren't doing an exhaustive dive I think it's okay if we shorten this to something like "OpenAI API's Models". 

```suggestion
# OpenAI API's Models
```